src/host/outputStream.cpp

```diff
@@ -33,6 +33,14 @@ ConhostInternalGetSet::ConhostInternalGetSet(_In_ IIoProvider& io) :
 // - <none>
 void ConhostInternalGetSet::ReturnResponse(const std::wstring_view response)
 {
+    auto& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
+
+    // ConPTY should not respond to requests. That's the job of the terminal.
+    if (gci.GetVtIo(nullptr))

```

Comment:
This could be replaced with `IsConsolePty()`.

Reply:
I've removed `IsConsolePty()` entirely now. The only places we called it were in `AdaptDispatch` and since ConPTY won't actually return responses anymore, it was indeed possible to remove all of them (at least from what I could tell).

Reply:
I get removing it from `ITerminalApi`, but I thought it was still being used in a couple of places in the `ConhostInternalGetSet` class itself (right here would have been one such case). Not a big deal if you prefer to get rid of it completely - I just thought it was a little more readable than the code above.

Reply:
> [...] I thought it was still being used in a couple of places in the `ConhostInternalGetSet` class itself [...]

Only two, actually! I've added a new `gci.IsConPTY()` getter which hopefully should make this a bit clearer. Maybe I should call it `IsConsolePty` like it was before...
There was one more in `ConhostInternalGetSet::ResizeWindow`, but I don't think that one was still relevant.

> I just thought it was a little more readable than the code above.

Yeah, I agree. However, I expect the entire ConPTY architecture to still change significantly in the future as we implement the 2nd goal here: https://github.com/microsoft/terminal/blob/dev/lhecker/13000-spec/doc/specs/%2313000%20-%20In-process%20ConPTY.md#goal-2-move-console-api-implementations-to-server
That should give us a ton of opportunity to make it not just more readable but also simpler.